gas-report.txt

```diff
@@ -0,0 +1,95 @@
+·------------------------------------------------|---------------------------|--------------|-------------------------------------·
+|              Solc version: 0.6.11              ·  Optimizer enabled: true  ·  Runs: 1500  ·  Block limit: 9007199254740991 gas  │
+·················································|···························|··············|······································
+|  Methods                                                                                                                        │
+··················|······························|·············|·············|··············|···················|··················
+|  Contract       ·  Method                      ·  Min        ·  Max        ·  Avg         ·  # calls          ·  eur (avg)      │
```

Comment:
Does this need to be commited? Most likely will change with every commit

Reply:
> Does this need to be committed?

It depends, between each PR we can track how much costs are being increased with each change, it's kind of keep track of gas usage between different commits.

But we can decide whether we want to keep it or not.

Reply:
Yes, that's something we'll have to setup as part of the CI/CD. Using a file in the git tree would be a very poor replacement.